corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -289,27 +289,49 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
         });
     };
 
+    const onlyVisibleColumns = function (detailObject) {
+        const indices = _.chain(detailObject.styles)
+            .map((value, index) => [value, index])
+            .filter(([s]) => s.displayFormat !== constants.FORMAT_ADDRESS_POPUP)
+            .map(([, index]) => index)
+            .value();
+
+        return {
+            styles: _.map(indices, index => detailObject.styles[index]),
+            headers: _.map(indices, index => detailObject.headers[index]),
+            tiles: _.map(indices, index => detailObject.tiles[index]),
+            details: _.map(indices, index => detailObject.details[index]),
+        };
+    };
+
     // return a case tile from a detail object (for persistent case tile and case tile in case detail)
     var getCaseTile = function (detailObject) {
+        var {
+            styles,
+            headers,
+            tiles,
+            details,
+        } = onlyVisibleColumns(detailObject);
         var detailModel = new Backbone.Model({
-            data: detailObject.details,
+            data: details,
             id: 0,
         });
         var numEntitiesPerRow = detailObject.numEntitiesPerRow || 1;
         var numRows = detailObject.maxHeight;
         var numColumns = detailObject.maxWidth;
         var useUniformUnits = detailObject.useUniformUnits || false;
-        var caseTileStyles = views.buildCaseTileStyles(detailObject.tiles, detailObject.styles, numRows,
+
+        var caseTileStyles = views.buildCaseTileStyles(detailObject.tiles, styles, numRows,
```

Comment:
overlooked the `detailObject.tiles` to `tiles` change here

Reply:
Good catch. Thanks